SSA/Projects/InstCombine/Base.lean

```diff
@@ -99,7 +99,13 @@ def reprWithoutFlags (op : MOp.BinaryOp) (prec : Nat) : Format :=
     | .udiv ⟨false⟩        => "udiv"
     | .udiv ⟨true⟩         => "udiv exact"
   Repr.addAppParen (Format.group (Format.nest
-    (if prec >= max_prec then 1 else 2) f!"InstCombine.MOp.BinaryOp.{op}"))
+    (if prec >= max_prec then 1 else 2) f!"llvm.{op}"))
+/- Changed this to use the "llvm" prefix for better pretty-printing.
+This affects every use of `Repr` for `MOp.BinaryOp`, which may not be desirable.
+TODO: Consider whether this change is acceptable globally.
+An alternative would be to introduce a separate `Serialize` typeclass that is same as `Repr` but uses the "llvm.{op}" format,
+while keeping `Repr` more verbose "InstCombine.MOp.BinaryOp.{op}". -/
```

Comment:
As mentioned on Zulip: for `Repr` there is the expectation that the result is valid Lean code, so this is indeed not acceptable. However, `ToString` has no such expectation, so if you define a `ToString` instance like this and use that for pretty printing, that is totally fine!